src/Apps/Search/Routes/Artists/SearchResultsArtists.tsx

```diff
@@ -15,14 +16,24 @@ export interface Props {
   location: Location
 }
 
+interface State extends LoadingAreaState {
+  page: number
+}
+
 const PAGE_SIZE = 10
 
-export class SearchResultsArtistsRoute extends React.Component<
-  Props,
-  LoadingAreaState
-> {
+export class SearchResultsArtistsRoute extends React.Component<Props, State> {
```

Comment:
Basically this adds the `page: $page` support for fetching the right data based on the incoming URL params, and then we added a little bit of local state to keep track of that for subsequent URL updates.